health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UploadDataCustom.js

```
@@ -0,0 +1,1188 @@
+
+    // Translate properties keys and their 'name' fields
+    Object.keys(schema?.properties).forEach((key) => {
+      const translatedKey = t(key);
+      const translatedProperty = { ...schema.properties[key], name: t(schema.properties[key].name) };
+      newProp[translatedKey] = translatedProperty;
+    });
+    const newRequired = schema?.required.map((e) => t(e));
+
+    newSchema.properties = newProp;
+    newSchema.required = newRequired;
+    delete newSchema.unique;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)